# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/indent/gcc40.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

This fixes the error:
gcc-40 -DHAVE_CONFIG_H -DLOCALEDIR=\"/usr/share/locale\" -I. -I. -I.. -I. -I. -I../intl    -g -O2 -c `test -f 'output.c' || echo './'`output.c
output.c:26: error: static declaration of 'output' follows non-static declaration
indent.h:100: error: previous declaration of 'output' was here

--- ./src/output.c~	2002-12-12 18:36:49.000000000 +0100
+++ ./src/output.c	2005-10-20 13:31:51.000000000 +0200
@@ -23,7 +23,7 @@
 
 RCSTAG_CC ("$Id: output.c,v 1.5 2002/12/12 17:36:49 david Exp $");
 
-static FILE            * output       = NULL;
+FILE            * output       = NULL;
 static BOOLEAN           inhibited    = 0;
 static buf_break_st_ty * buf_break_list = NULL;
 
